LibertyDSNP/example-client
# style from title and metadata of posts and replies

Purpose
---------------
feature
[https://www.pivotaltracker.com/story/show/179649000](https://www.pivotaltracker.com/story/show/179649000)
create prettier UI and fix timestamp bug


Solution
---------------
- change styling for reply and post from title
- add the ability to click on users' profile from replies
- convert timestamp from unix bigint to ISO when you call addFeedItem
- move the timestamp to the top right corner

Change summary
---------------
* described in solution


Steps to Verify
----------------
1. Run code
2. create new post
3. see that it looks good and there are no errors


Additional details / screenshot
----------------
- Any supplemental pictures or material
- ![Screenshot]()
